src/common/utils/fetch-and-deserialize.ts

```diff
@@ -0,0 +1,25 @@
+import { WorkOS } from '../../workos';
+import { List, ListResponse, PaginationOptions } from '../interfaces';
+import { deserializeList } from '../serializers';
+
+function setDefaultOptions(options?: PaginationOptions): PaginationOptions {
+  return {
+    ...options,
+    order: options?.order || 'desc',
+  };
+}
+
+export async function fetchAndDeserialize<T, U>(
+  workos: WorkOS,
+  endpoint: string,
+  deserializeFn: (data: T) => U,
+  options?: PaginationOptions,
+): Promise<List<U>> {
+  const finalOptions = setDefaultOptions(options);
```

Comment:
Just a styling bit, I don't think we need to set this in another variable. Could just be `query: setDefaultOptions(options)` on line 21.